operations/app/src/modules/key_vault/main.tf

```diff
@@ -101,6 +102,15 @@ resource "azurerm_key_vault_access_policy" "frontdoor_access_policy" {
   certificate_permissions = [ "Get" ]
 }
 
+resource "azurerm_key_vault_access_policy" "terraform_access_policy" {
+    key_vault_id = azurerm_key_vault.application.id
+    tenant_id = data.azurerm_client_config.current.tenant_id
+    object_id = local.terraform_object_id
+
+    secret_permissions = [ "Get" ]
+    key_permissions = [ "Get" ]
```

Comment:
Why does terraform get `key_permissions = ["Get"]` on this vault but not on the app_config vault?

Reply:
Key permission is used to reference the HSM key for the Storage Account and Postgres database